### DISCUSSION TITLE
WinForms - Unable to get mouse wheel or key presses

### DISCUSSION THREAD

**Discussion by EricBourque** [cefsharp/CefSharp#4364] [General]
Dear CefSharp team,

I am using CefSharp.WinForms 108.4.120
I am trying to use CTRL-MouseScroll to change the zoom level, I can't seem to get the mouse scroll value when the browser is focused.
The F5 key is also decreasing the width of my URL textbox every I press it for some reason.

How can I use CTRL + MouseScroll when the browser is focused (I am going to use this for the zoom level)

**Comment by amaitland** [comment]
This is the expected behaviour as `Chromium` doesn't propagate the low level messages.

See https://github.com/cefsharp/CefSharp/issues/3340

> How can I use CTRL + MouseScroll when the browser is focused (I am going to use this for the zoom level)

You should be able to adapt the example at https://github.com/cefsharp/CefSharp/blob/cefsharp/109/CefSharp.WinForms.Example/BrowserTabUserControl.cs#L363 to handle https://learn.microsoft.com/en-us/windows/win32/inputdev/wm-mousewheel
